src/Lights/Shadows/shadowGenerator.ts

```diff
@@ -1514,6 +2090,12 @@ export class ShadowGenerator implements IShadowGenerator {
             this._shadowMap.dispose();
             this._shadowMap = null;
         }
+        if (this.useCSM) {

```

Comment:
every resources should be disposed

Reply:
ok , changed to 
```
if (this.useCSM) {
		for (let i = 0; i < this._cascadeShadowMaps.length; i++){
			this._cascadeShadowMaps[i].dispose();
		}
		this._viewSpaceFrustums = [];
		this._viewSpaceBoundingSpheres = [];
            this._CSMtransformMatrices = [];
        }
```
_cascadeShadowMaps are no longer nullable, cf, above fix. BoundingSphere, Matrix, Vector3 cannot be disposed nor set to null. Dunno what bjs standard for dispose should be, lemme know if this isn't sufficient.

Reply:
Only the GL ones need to be disposed or the ones retaining memory. it is perfect like this.